src/test/resources/cucumber/features/integrationTests/pull-request-comments-update.feature

```diff
@@ -35,7 +35,6 @@
     Scenario Outline: Pull request arrives from ADO to CxFlow, then scan is initiated, pull request should be without comments, and we should verify that the comments are new.
```

Comment:
Here the scenario's title is basically the rewriting of the scenario steps. Can the title be shorter?